websites/G/GameBanana/presence.ts

```diff
@@ -60,6 +60,14 @@ presence.on("UpdateData", async () => {
 			"Support",
 			"Studio",
 			"Wiki",
+		],
+		gamesettingspages = [
+			"welcomemessage",
+			"subcontrols",
+			"rules",
+			"attribs",
+			"details",
+			"uberstyle",
 		];
```

Comment:
all of these variables are never modified, you should declare them at the top level to save some performance

Reply:
What do you mean by "the top level"?

Reply:
I mean it should be declared in the global scope as opposed to inside the event. And yes, those too